src/Utilities/CssConcatenator.php

```diff
@@ -102,12 +104,18 @@ public function getCss(): string
      * @param string $media The media query for rules to be appended, e.g. "@media screen and (max-width:639px)",
      *        or an empty string if none.
      *
-     * @return \stdClass Object with properties as described for elements of `$mediaRules`.
+     * @return object{
+     *   media: string,
+     *   ruleBlocks: array<int, object{
+     *     selectorsAsKeys: array<string, array-key>,
+     *     declarationsBlock: string
+     *   }>
+     * }
      */
-    private function getOrCreateMediaRuleToAppendTo(string $media): \stdClass
+    private function getOrCreateMediaRuleToAppendTo(string $media): object
     {
         $lastMediaRule = \end($this->mediaRules);
-        if ($lastMediaRule instanceof \stdClass && $media === $lastMediaRule->media) {
+        if (\is_object($lastMediaRule) && $media === $lastMediaRule->media) {
```

Comment:
Ditto.

Reply:
This one can actually be retained using `instanceof \stdClass`, presumably because Psalm does not need to be able to determine the type of `$lastMediaRule->media` because it is only used as an operand to `===`.